Tests/Transfer/HyperwalletTransferTests.swift

```diff
@@ -138,6 +138,30 @@ class HyperwalletTransferTests: XCTestCase {
         XCTAssertNotNil(statusTransitionResponse, "The `statusTransitionResponse` should not be nil")
         verifyStatusTransitionResponse(statusTransitionResponse)
     }
+
+    func testListTransfer_success() {
```

Comment:
update name to testListTransfers
add testListTransfers_emptyResult similar to testListBankAccounts_emptyResult

Reply:
Renamed. Added `testListTransfers_emptyResult` test